tools/openshift/backend.dc.yaml

```diff
@@ -78,12 +72,56 @@ objects:
                 limits:
                   cpu: '${MAX_CPU}'
                   memory: '${MAX_MEM}'
+              volumeMounts:
+                # - name: tls-certs
```

Comment:
Can we just remove tls-certs? They won't ever be needed as we have the -tls specific template.

Reply:
Yes :D